package.json

```diff
@@ -33,7 +33,8 @@
     "express-prismic": "^1.1.1",
     "graphql": "^0.6.0",
     "lodash": "^4.13.1",
```

Comment:
My suspicion is that Lodash is an **implicit** Babel dependency that another module provides **explicitly**. Babel are known to monkey patch other libs dependencies